packages/core/cms/sections.json

```diff
@@ -233,18 +767,39 @@
             "properties": {
               "key": {
                 "title": "Key",
-                "description": "Tip: For collections, use: productClusterIds",
+                "hint": "For collections, use: productClusterIds",
                 "type": "string",
                 "default": "productClusterIds"
               },
               "value": {
                 "title": "Value",
-                "description": "Tip: For collection 140, use: 140",
+                "hint": "For collection 140, use: 140",
                 "type": "string",
                 "default": "140"
               }
             }
           }
+        },
+        "product-card-configuration": {
+          "title": "Product Card Configuration",
+          "type": "object",
+          "properties": {
+            "show-discount-badge": {
+              "title": "Show discount badge?",
+              "type": "boolean",
+              "default": true
+            },
+            "show-buy-button": {
+              "title": "Show buy button?",
+              "type": "boolean",
+              "default": true
+            },
```

Comment:
I think we don't have this checking in frontend. 👀

Reply:
We control this behavior using the OnClick Button method, who to integrate this component can improve